src/app/core/entity-details/form/form.component.ts

```diff
@@ -32,28 +34,49 @@ import { FieldGroup } from "./field-group";
   ],
   standalone: true,
 })
-export class FormComponent<E extends Entity> implements FormConfig, OnInit {
+export class FormComponent<E extends Entity>
+  implements FormConfig, OnInit, OnDestroy
+{
   @Input() entity: E;
   @Input() creatingNew = false;
 
   @Input() fieldGroups: FieldGroup[];
 
+  filteredFieldGroups: FieldGroup[];
+
   form: EntityForm<E>;
 
+  private _destroyed = new Subject<void>();
+
   constructor(
     private router: Router,
     private location: Location,
     private entityFormService: EntityFormService,
     private alertService: AlertService,
+    private abilityService: AbilityService,
   ) {}
 
+  ngOnDestroy(): void {
+    this._destroyed.next();
+    this._destroyed.complete();
```

Comment:
We use the [@ngneat/until-destroy](https://www.npmjs.com/package/@ngneat/until-destroy) package to make the unsubscribing a little easier (e.g. see the `NoteDetailsComponent` on how to use it)

Reply:
done